config/software/git-windows.rb

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2016 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "git-windows"
+default_version "2.8.1"
+
+arch_suffix = windows_arch_i386? ? '32' : '64'
+source url: "https://github.com/git-for-windows/git/releases/download/v#{version}.windows.1/PortableGit-#{version}-#{arch_suffix}-bit.7z.exe"
+
+if windows_arch_i386?
+  version("2.8.1") { source sha256: "0b6efaaeb4b127edb3a534261b2c9175bd86ee8683dff6e12ccb194e6abb990e" }
+else
+  version("2.8.1") { source sha256: "dc9d971156cf3b6853bc0c1ad0ca76f1d2c24478cca80036919f12fe46acd64e" }
+end
+
+build do
+
+  env = with_standard_compiler_flags(with_embedded_path)
+
+  source_7z = "#{project_dir}/PortableGit-#{version}-#{arch_suffix}-bit.7z.exe"
+  destination = "#{install_dir}/embedded/git"
+
+  command "#{source_7z} -y"
+  sync "PortableGit", "#{windows_safe_path(destination)}", env: env
+
+  block "Create bat files to point to executables under embedded/git/cmd" do
```

Comment:
Can you explain what these `*.bat` files do? Might be worth adding a comment so others are read in.